### PR TITLE
Talos - Bump @bbc/psammead-assets

### PR DESCRIPTION
**Pull request by bbc-news-frameworks**
👋 The following packages have been updated:

@bbc/psammead

<details>
<summary>Details</summary>
@bbc/psammead-assets  ^2.11.0  →  ^2.12.0

| Version | Description |
|---------|-------------|
| 2.12.0 | [PR#2935](https://github.com/bbc/psammead/pull/2935) Add a Clock icon to core icons |
</details>

@bbc/psammead-image-placeholder

<details>
<summary>Details</summary>
@bbc/psammead-assets  ^2.11.0  →  ^2.12.0

| Version | Description |
|---------|-------------|
| 2.12.0 | [PR#2935](https://github.com/bbc/psammead/pull/2935) Add a Clock icon to core icons |
</details>

@bbc/psammead-embed-error

<details>
<summary>Details</summary>
@bbc/psammead-assets  ^2.11.0  →  ^2.12.0

| Version | Description |
|---------|-------------|
| 2.12.0 | [PR#2935](https://github.com/bbc/psammead/pull/2935) Add a Clock icon to core icons |
</details>

@bbc/psammead-navigation

<details>
<summary>Details</summary>
@bbc/psammead-assets  ^2.11.0  →  ^2.12.0

| Version | Description |
|---------|-------------|
| 2.12.0 | [PR#2935](https://github.com/bbc/psammead/pull/2935) Add a Clock icon to core icons |
</details>

@bbc/psammead-media-indicator

<details>
<summary>Details</summary>
@bbc/psammead-assets  ^2.11.0  →  ^2.12.0

| Version | Description |
|---------|-------------|
| 2.12.0 | [PR#2935](https://github.com/bbc/psammead/pull/2935) Add a Clock icon to core icons |
</details>

@bbc/psammead-play-button

<details>
<summary>Details</summary>
@bbc/psammead-assets  ^2.11.0  →  ^2.12.0

| Version | Description |
|---------|-------------|
| 2.12.0 | [PR#2935](https://github.com/bbc/psammead/pull/2935) Add a Clock icon to core icons |
</details>

@bbc/psammead-bulletin

<details>
<summary>Details</summary>
@bbc/psammead-assets  ^2.11.0  →  ^2.12.0

| Version | Description |
|---------|-------------|
| 2.12.0 | [PR#2935](https://github.com/bbc/psammead/pull/2935) Add a Clock icon to core icons |
</details>

@bbc/psammead-media-player

<details>
<summary>Details</summary>
@bbc/psammead-assets  ^2.11.0  →  ^2.12.0

| Version | Description |
|---------|-------------|
| 2.12.0 | [PR#2935](https://github.com/bbc/psammead/pull/2935) Add a Clock icon to core icons |
</details>

### CODE REVIEW

File: packages/components/psammead-story-promo/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 4.0.0-alpha.2 | [PR#2989](https://github.com/bbc/psammead/pull/2989) Update screenshot |
```
Comment (Bopchy) [inline packages/components/psammead-story-promo/CHANGELOG.md]:
```suggestion
| 4.0.0-alpha.2 | [PR#2989](https://github.com/bbc/psammead/pull/2989) Update snapshot |
```

**Review by Bopchy — COMMENTED** [review]
Other than the suggestion above, everything looks good 👍

File: packages/components/psammead-story-promo/CHANGELOG.md (hunk 1 above)
Comment (Bopchy) [reply packages/components/psammead-story-promo/CHANGELOG.md]: I'm guessing this should be `snapshot` ?

Comment (j-pendlebury) [reply packages/components/psammead-story-promo/CHANGELOG.md]: oops 🙈  you're right

**Review by Bopchy — APPROVED** [review]

**Review by j-pendlebury — APPROVED** [review]

**Pull request merged**